src/Sulu/Bundle/AdminBundle/Resources/js/components/Modal/ModalBox.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import React from 'react';
+import keydown from 'react-keydown';
+import Icon from '../Icon';
+import type {ModalProps} from './types';
+import Actions from './Actions';
+import modalBoxStyles from './modalBox.scss';
+
+type Props = ModalProps & {
+    onRequestClose: () => void,
+}
+
+const CLOSE_ICON = 'times';
+
+export default class ModalBox extends React.PureComponent<Props> {
+    static defaultProps = {
+        actions: [],
+    };
+
+    @keydown('esc') requestClose() {
+        this.props.onRequestClose();
+    }
+
+    render() {
+        const {title, onRequestClose, children, actions, onConfirm, confirmText} = this.props;
+
+        return (
+            <section className={modalBoxStyles.box}>
+                <header>
+                    {title}
+                    <Icon name={CLOSE_ICON} className={modalBoxStyles.icon} onClick={onRequestClose} />
```

Comment:
use `requestClose` resp. `handleRequestClose` directly? What if we have to do more than just calling one callback?